File: LeetCode/Remove Duplicates from Sorted Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        vector<int> ans;
        int i=0, j=1;
        ans.push_back(nums[0]);
        int k=1;
        while(j<n){
            if(nums[i]==nums[j]){
                j++;
            }else{
                i=j;
                ans.push_back(nums[i]);
                k++;
            }
        }

        for(int i=0; i<k; i++){
            nums[i]= ans[i];
        }
         return k;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        left = 0
        right = 1
        countUnique = 1

        while right < n:
            if nums[right - 1] == nums[right]:
                right += 1

            else:
                left += 1
                nums[left] = nums[right]
                right += 1
                countUnique += 1

        left = left + 1
        while left < n:
            nums[left] = '_'
            left += 1

        return countUnique

            